dist/debian/control.template

```diff
@@ -0,0 +1,13 @@
+Source: %{product}-cqlsh
+Section: misc
+Priority: extra
+Maintainer: Israel Fruchter <fruch@scylladb.com>
+Homepage: http://www.scylladb.com
+Standards-Version: 3.9.5
+Rules-Requires-Root: no
+
+Package: %{product}-cqlsh
+Architecture: all
+Depends: python3
+Conflicts: cassandra
+Description: cqlsh is a Python-based command-line client for running CQL commands on a cassandra cluster.
```

Comment:
When we splited scylla-tools package into two (scylla-tools and scylla-tools-core), we added Replaces and Breaks on debian/control to avoid conflict while upgrading packages, like this:
https://github.com/scylladb/scylla-tools-java/commit/59531395df77dc19da5638df336a1e8dcadf0d6c
(It fixes scylladb/scylladb#2732 and scylladb/scylladb#2765)

I guess we need similar one here, something like:
```
Replaces: scylla-tools (<< 5.2~rc0)
Breaks: scylla-tools (<< 5.2~rc0)
```

Note that I haven't tested this yet, we need to verify upgrade works with scylla-cqlsh, and make sure it's not conflict with older scylla-tools.

Reply:
add this change, and tested it, it's works as expected (installed 5.1 scylla-tools, and then scylla-cqlsh)